front_end/panels/ai_chat/ui/settings/components/SettingsHeader.ts

```
@@ -0,0 +1,35 @@
+ */
+export function createSettingsHeader(
+  container: HTMLElement,
+  onClose: () => void,
+): HTMLElement {
+  const headerDiv = document.createElement('div');
+  headerDiv.className = 'settings-header';
+  container.appendChild(headerDiv);
+
+  const title = document.createElement('h2');
+  title.className = 'settings-title';
+  title.textContent = i18nString({ settings: 'Settings' }.settings);
```

ESLint

[error] Calling i18nString/i18nLazyString requires a UIStrings member as the first argument (e.g., UIStrings.someString).

(rulesdir/l10n-i18nString-call-only-with-uistrings)